packages/pn-pa-webapp/src/redux/notification/__test__/reducers.test.ts

```diff
@@ -148,4 +149,9 @@ describe('Notification detail redux state tests', () => {
     expect(state.legalFactDownloadRetryAfter).toEqual(0);
     expect(state.legalFactDownloadUrl).toEqual('');
   });
+
+  it('Should be able to cancel notification', async () => {
+    const action = await store.dispatch(cancelNotification('mocked-iun'));
+    expect(action.type).toBe('cancelNotification/rejected');
```

Comment:
why the rejected check?

Reply:
I replace with fullfiled and skip the test